common/src/main/java/org/neo4j/cypher/export/CypherResultSubGraph.java

```diff
@@ -44,12 +47,16 @@ public void add(Relationship rel) {
     public static SubGraph from(Transaction tx, Result result, boolean addBetween) {
         final CypherResultSubGraph graph = new CypherResultSubGraph();
         final List<String> columns = result.columns();
-        result.forEachRemaining(row -> {
-            for (String column : columns) {
-                final Object value = row.get(column);
-                graph.addToGraph(value);
-            }
-        });
+        try {
+            result.forEachRemaining(row -> {
+                for (String column : columns) {
+                    final Object value = row.get(column);
+                    graph.addToGraph(value);
+                }
+            });
+        } catch (AuthorizationViolationException e) {
+            throw new RuntimeException(INVALID_QUERY_MODE_ERROR);
```

Comment:
Besides `apoc.export.cypher.query` and some procedures which does not take queries,  this method is also used by `apoc.export.graphml.query`. Do we want to add a test for that one as well?

Reply:
Ah I missed one! nice catch :)